### PR TITLE
Record whether a CodeAction is a fix or not

### PR DESCRIPTION
**Pull request by 333fred**
Omnisharp-roslyn side of fixing https://github.com/OmniSharp/omnisharp-vscode/issues/5335.

### CODE REVIEW

File: src/OmniSharp.Roslyn.CSharp/Services/Refactoring/V2/AvailableCodeAction.cs [hunk 1]
```diff
@@ -10,11 +10,13 @@ public class AvailableCodeAction
     {
         public CodeAction CodeAction { get; }
         public CodeAction ParentCodeAction { get; }
+        public bool IsCodeFix { get;  }
```
Comment (AlFasGD) [inline src/OmniSharp.Roslyn.CSharp/Services/Refactoring/V2/AvailableCodeAction.cs:13]: Overlooked double space

**Review by AlFasGD — COMMENTED** [review]
Not too sure about the choice of using tuples of the form `(CodeAction, bool)` instead of a record like `CodeActionInfo(CodeAction Action, bool IsCodeFix)` (not the best name I came up with)

**Comment by 333fred** [comment]
Tuples are great for these types of intermediate structure scenarios.

**Comment by 333fred** [comment]
@JoeRobich @filipw for review please.

**Review by JoeRobich — APPROVED** [review]
Thanks!

**Pull request merged**